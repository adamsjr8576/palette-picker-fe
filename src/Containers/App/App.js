import React from 'react';
import './App.scss';
import Header from '../../Components/Header/Header';
import PaletteForm from '../PaletteForm/PaletteForm';

const App = () => {
  return (
    <main className="App">
      <Header />
      <PaletteForm />
      
    </main>
  );
}

export default App;
