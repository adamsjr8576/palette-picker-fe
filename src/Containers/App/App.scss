.App {
  display: flex;
  flex-wrap: wrap;
  text-align: center;
  height: 95vh;
}

.section-project {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-between;
  width: 30%;
}
