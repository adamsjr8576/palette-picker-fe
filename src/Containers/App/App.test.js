import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import { shallow } from 'enzyme';

describe('App', () => {
  let wrapper;
  
  describe('App Component', () => {

    it('should match the snapshot', () => {
      wrapper = shallow(<App />)

      expect(wrapper).toMatchSnapshot()
    });
  });

});
