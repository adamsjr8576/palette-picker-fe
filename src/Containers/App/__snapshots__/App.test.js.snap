// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App Component should match the snapshot 1`] = `
<main
  className="App"
>
  <Connect(PaletteGenerator) />
  <section
    className="section-project"
  >
    <Connect(Header) />
    <Connect(ProjectForm) />
    <Connect(PaletteForm) />
  </section>
  <Connect(ProjectContainer) />
</main>
`;
